app/utils.py

```diff
@@ -80,10 +80,11 @@ class NextUrlSanitizer:
     def sanitize(url: Optional[str], allowed_domains: List[str]) -> Optional[str]:
         if not url:
             return None
-        result = urllib.parse.urlparse(url)
+        replaced = url.replace("\\", "/")
```

Comment:
I wonder if there's a library in python that can validate an url so we only need to call `urllib.parse.urlparse(replaced)` if the url is valid.